modules/Bio/EnsEMBL/Compara/RunnableDB/Flatfiles/UpdateTableFromFile.pm

```diff
@@ -27,7 +27,7 @@ column names (`primary_key` is required and should be present in all files).
 
 =cut
 
-package Bio::EnsEMBL::Compara::RunnableDB::UpdateTableFromFile;
+package Bio::EnsEMBL::Compara::RunnableDB::Flatfiles::UpdateTableFromFile;
```

Comment:
This module is not used any more. I imagine the changes you've made in it were for a previous iteration of your work, but without any context, I can't assess why these changes are required. I'll just approve it as it is